src/intent.html

```diff
@@ -221,20 +221,14 @@ <h3 id="mixing_intent_properties">Intent Properties</h3>
    but as with any properties, is by default ignored but may have a
    system-specific effect.</p>
 
-    <p>As with [=Concepts=], The Working group maintains two lists
-      of [=property=] values.</p>
+    <p>Similar to the lists of [=Concepts=], The Working Group maintains a list of [=property=] values that should be implemented.</p>
     <ul>
-      <li><dfn id="intent-core-properties">Core properties</dfn>: This is a
-      <a href="https://w3c.github.io/mathml-docs/intent-core-properties">list core of properties</a>
+      <li><dfn id="intent-core-properties">Properties list</dfn>: This is the full
+      <a href="https://w3c.github.io/mathml-docs/intent-core-properties">list of properties</a>
       maintained by the Math Working Group</li>
-       <li><dfn id="intent-open-properties">Open properties</dfn>: This is an
-      <a href="https://w3c.github.io/mathml-docs/intent-open-properties">open list of properties</a>
-      with contributions welcome from the community.
-      Implementors of MathML systems that implement additional properties are encouraged
-      to make a pull request to add them to the list of  [=Open Properties=].</li>
     </ul>
-   <p>The definitive list of [=Core Properties=] is maintained at
-   Github. Here, we describe the major classes of property affecting speech generation
+   <p>The definitive  [=Properties list=] is maintained at
```

Comment:
too much space

Reply:
well it;s html so the space does no harm, but I cleanred up (and a spurious "below" was left in the sentence from the original form (checkin to follow)